bevy_kayak_renderer/src/render/unified/pipeline.rs

```diff
@@ -337,6 +337,9 @@ struct QuadVertex {
 #[derive(Copy, Clone, ShaderType)]
 struct QuadType {
     pub t: i32,
+    pub _padding_1: i32,
```

Comment:
Another option here is to use IVec3 like:
```rust
struct QuadType {
    pub t: i32,
    pub _padding: iVec3,
}
```

Reply:
That would be a lot cleaner - looking back through the docs, I was reading the `AlignOf` column and not the `SizeOf` column for the different types, so thought the `vec3` types were all 16 bytes